### ISSUE TITLE
DoTogglePause or DoPause not working anymore with rigidbody2d

### ISSUE THREAD

**Issue by FedericoCara** [Demigiant/dotween#81]
I'm trying to make an older project work again. It was made in Unity 4.6. The problem is with a tween that does DoMoveY to a rigidbody2D. When I pause the game I do rigidbody2D.togglepause(), and the rigidbody2D continue moving (this had never happened before). What's also weird, is that when I resume the tween, it comes back to the position where it was when I paused it, and continues the moving correctly.
Some details about what has changed: now I'm using Windows 10 instead of Windows 7; also I have installed Unity 5.3.5f1 (but I'm opening the project with Unity 4.6) on the same pc so the monodevelop version should be different too.
I try using dotween last version super compatible too but it didn't make any difference.

**Comment by Demigiant** [comment]
Hi,

If the tween resumes from where you paused it, it means it's 100% paused, so there are two possible things happening:
A) You have multiple tweens on the same object, so one is continuing while the other pauses, and when you resume the first one it overwrites the second one
B) Some physics2D behaviour changed in Unity, and after you pause the tween the physics take control and continue to move the object using inertia

**Comment by FedericoCara** [comment]
Thanks for the quick answer. A) There are no other tweens. B) Yes, it probably has to do with the rigidbody, if I manually set the rigidbody to kinematic it stops moving, but then the tween doesn't resume correctly. However, this must have something to do with unity version, win version or .net version since I've tried many commits along the project and they all have the same bug. This afternoon I'll give it a try in my old pc with Win7 to see if it works correctly.
